docs/integrations/sources/amazon-ads.md

```diff
@@ -9,6 +9,7 @@ This page contains the setup guide and reference information for the Amazon Ads
 * Region
 * Start Date (Optional)
 * Profile IDs (Optional)
+* Marketplace IDs (Optional)
```

Comment:
Add a note here or in the setup explaining how it works using profiles and marketplace ids